build.gradle

```diff
@@ -162,7 +162,7 @@ subprojects {
 }
 
 task myWrapper(type: Wrapper) {
```

Comment:
I think nobody uses this anymore, I think the whole task should be removed

Reply:
Good idea - it's confusing to have this task here